pkg/config/config_template.yaml

```
@@ -1193,6 +1193,13 @@ api_key:
   #
   # file_wildcard_selection_mode: `by_name`
 
+  ## @param log_line_max_size - integer - optional - default: 256
+  ## @env DD_LOG_LINE_MAX_SIZE - integer - optional - default : 256
+  ## The maximum size of single log in kilobytes.
+  ## If a line is bigger than this limit, it will be truncated.
```

Since we don't have units on this value, it may be good to be explicit about the expectations from this setting:

```suggestion
  ## If a log line is larger than this byte limit, the overflow bytes will be truncated.
  
  ## Please note that depending on the log's content encoding, the final
  ## truncated character count may vary.
  
```

Alternatively, maybe we can just name the variable `log_line_max_bytes`? 🤷

---

Oops, just changed it to `max_message_size`. Are we keeping the `log` prefix? If not, `max_message_size_bytes`?

---

I'd say drop the log prefix since we already have the `logs_confg.` prefix. And i'd be ok with `max_message_size_bytes`